routes/utils/errors.js

```diff
@@ -10,6 +10,12 @@ function hoodieError (options) {
   return error
 }
 
+module.exports.NO_ACTIVE_SESSION = hoodieError({
+  name: 'Unauthorized',
+  message: 'No active session',
+  status: 401
+})
```

Comment:
I’d call the error `INVALID_SESSION`, and the message `session invalid`. 

[In the spec](http://docs.accountjsonapi.apiary.io/#reference/current-user/profile/fetch) we have

```
{
    errors: [{
        "status": "401",
        "title": "Unauthorized",
        "detail": "Session expired"
    }]
}
```

I’d update that too. Invalid is more accurate than expired, as there can be more reason than a session timeout. What do you think?